.github/workflows/issue-opened.yml

```diff
@@ -75,19 +77,21 @@ jobs:
                     labelExists = true;
                   } catch {}
 
-                  if (labelExists) {
-                    // Check if it's an unsupported major
-                    const { ElectronVersions } = await import('${{ github.workspace }}/node_modules/@electron/fiddle-core/dist/index.js');
-                    const versions = await ElectronVersions.create(undefined, { ignoreCache: true });
+                  const { ElectronVersions } = await import('${{ github.workspace }}/node_modules/@electron/fiddle-core/dist/index.js');
+                  const electronVersions = await ElectronVersions.create(undefined, { ignoreCache: true });
+                  const validVersions = [...electronVersions.supportedMajors, ...electronVersions.prereleaseMajors];
 
-                    const validVersions = [...versions.supportedMajors, ...versions.prereleaseMajors];
-                    if (validVersions.includes(major)) {
+                  if (validVersions.includes(major)) {
+                    if (labelExists) {
                       labels.push(versionLabel);
                     }
+                  } else {
+                    hasUnsupportedVersion = true;
```

Comment:
I _think_ the case would still cause the nightly to end up marked as an unsupported version, right, since it wouldn't be included in `validVersions`? Could we maybe check and see if the version string includes `nightly` or something to get around that?

Reply:
An easy fix could be to just put the label of the previous version on the issue so we'll pick it up at the WG releases meeting. For example, issues on 38 nightlies would get the 37 label.